Jenkinsfile

```diff
@@ -9,97 +9,9 @@ pipeline {
   }
 
   stages {
-    stage('Image Build') {
+    stage('Push Sidecar') {
```

Comment:
What happened here that a whole bunch of the Jenkinsfile is deleted?

Reply:
this was for me to push the test container :)